src/WebServiceProvider.php

```diff
@@ -412,4 +411,125 @@ private function configure_api()
             ])),
         ]);
     }
+
+    /**
+     * Return the plugin author EVE Character ID.
+     *
+     * @return int|null
+     */
+    public function getAuthorEveCharacterID(): ?int
+    {
+        return 1477919642;
+    }
+
+    /**
+     * Return the plugin author name (or any public nickname).
+     *
+     * @return string
+     */
+    public function getAuthorName(): string
+    {
+        return 'qu1ckkkk';
+    }
+
+    /**
+     * Return the plugin author e-mail address.
+     *
+     * @return string|null
+     */
+    public function getAuthorMailAddress(): ?string
+    {
+        return 'theninjabag@gmail.com';
```

Comment:
I dont think this information (getAuthorEveCharacterID(), getAuthorName() and getAuthorMailAddress(), getAuthorSlackNickname()) is useful. A github link is enough.